src/model.py

```diff
@@ -78,36 +78,27 @@ def main(input, output):
     print("Best score: %0.3f" % (random_search.best_score_))
 
     # Create optimized model
-    imb_pipeline = make_imb_pipeline(
-        RandomUnderSampler(random_state=21),
+    model = make_imb_pipeline(
+        RandomUnderSampler(),
         OneHotEncoder(handle_unknown="ignore", sparse=False),
         LogisticRegression(
             max_iter=2000,
-            C=random_search.best_params_["logisticregression__C"]
+            C = random_search.best_params_["logisticregression__C"]
         )
     )
 
     # Determine cross-validation scores of optimized model
     results["Logistic Regression Optimized"] = mean_std_cross_val_scores(
-        imb_pipeline, X_train, y_train, return_train_score=True
+        model, X_train, y_train, return_train_score=True
     )
     result_df = pd.DataFrame(results)
-    result_df.index.name = "score_type"
 
     # Create output tables/images
-    result_df.to_csv("results/score_results.csv")
+    save_df(result_df, "score_results", output)
 
-    # Creating the best model
-    model = LogisticRegression(
-        max_iter=2000,
-        C=random_search.best_params_["logisticregression__C"]
-    )
-
-    # Storing optimized model
-    pickle.dump(model, open(f"{output}final_model.rds", "wb"))
```

Comment:
I think you will need to retain the above script to create the 'model'

Reply:
delete the pickle.dump line though. Keep the one below this.

Reply:
its in line 100